Add documentation on how-to interpret the map (#179)
nextstrain/nextstrain.org: static-site/content/docs/07-visualisation/02-map-interpretation.md

---
title: "How to interpret what's shown on the map"
anchorText: "Interpreting the Map"
---

The map is a centerpiece of Nextstrain's visualisation, able to convey complex information to a wide audience.
This how-to is intended to provide an overview of what's being displayed, and hopefully dispel some potential misinterpretations.

![](./img/labelled-map.png)
_Figure 1. [Nextstrain map](https://nextstrain.org/zika) of Zika virus movement between countries. Numerical labels are referred to in the text._

Before we begin, it's important to keep in mind that all the data displayed on the map is drawn directly from the phylogenetic tree.

> In practice, every dataset is subject to some form of sampling bias, and therefore so will the map.

Reviewer: ```suggestion
In practice, every dataset is subject to some form of sampling bias, and therefore the map will be as well.
```

> If there are no samples (genomes) from a certain location in the dataset being visualised, then we have no data to displaye on our map!

Reviewer: ```suggestion
If there are no samples (genomes) from a certain location in the dataset being visualised, then we will have no data to display on our map!
```

> As you **interact with Nextstrain** -- whether you zoom into the tree, filter the data, change the color-by or geographic resolution -- then every panel, including the map, will update to represent just the selected data under the desired view settings.

Reviewer: ```suggestion
As you **interact with Nextstrain** -- whether you zoom into the tree, filter the data, change the color-by or geographic resolution -- then every panel, including the map, will update to represent just data selected under the current view settings.
```

## Demes (Circles)

Perhaps the most fundamental aspect of Nextstrain's map, the circles (demes) associate samples (tips on the tree) with locations on the map.
The demes use different visual techniques to convey different pieces of information:

![](./img/labelled-deme.png)
_Figure 2. Examples of demes.
B,C and D are representations of samples from the USA in [the Zika dataset](https://nextstrain.org/zika) with different choices of color-by._

> The **size (area) of the circle** is used to convey the number of samples associated with the current location.

Reviewer: Will this still be true once the latest changes from Richard are merged in? I think so, but wanted to check. If not, I think we can just be a little more vague and it will be fine :)

Author: Those changes are already up on nextstrain.org so I tried to use language to indicate that a deme's size "should only be interpreted relative to other circles currently displayed on the map". 

I think the only way to do this going forward is to add a dynamic legend showing the sizes.

Reviewer: I think this is good!

This should only be interpreted relative to other circles currently displayed on the map, and may change as you zoom into the map or subsample the data.
For instance, in Fig 2A, the larger circle represents a deme with four times as many samples as the smaller one.
As you filter the dataset, change the displayed time range, or zoom into the tree, the relative sizes of the circles will change according to what locations the in-view samples represent.

> Ancestral (inferred) nodes on the tree are _not_ represented by circles on the map.

Reviewer: I don't think it's possible to pass these in. It's certainly possible to have ancestors which haven o inferred *sequence*, but I don't know about not having inferred the internal nodes at some point. At most, I'd say this was a pretty edge case of some kind of weird tree drawn only using known... something.

> Ancestral (inferred) nodes on the tree are _not_ represented by circles on the map.

Reviewer: (It is possible to pass in a tree where the ancestors were inferred by some other program... but they'll still have been inferred at some point.)

> Ancestral (inferred) nodes on the tree are _not_ represented by circles on the map.

Author: It's a bit complicated.

In _Augur_ as it currently stands, sequences, traits and times of internal nodes will all be inferred. But there are some bioinformatic approaches which can place observed samples at internal nodes.

In _Auspice_ we try to be agnostic of how the data was generated (although it's not always possible).

> A circle's **color** conveys the metadata currently selected as the overall coloring of the dataset, and the way by which we convey this changes according to the type of color-scale employed.

Reviewer: ```suggestion
A circle's **color** conveys the metadata currently selected as the overall coloring of the dataset. The way we convey this changes according to the type of color-scale employed.
```
(I think it's good to have a clear one-sentence intro to the paragraph!)

For instance, in Fig 2B the circle comprises samples from the USA and is therefore colored red (this is actually the circle labelled "1" in Figure 1). 
In Fig 2C we have a discrete color scale representing authorship, and we convey this information by a pie chart with areas of each slice representing the proportion of samples in this location from each author.
In Fig 2D we have a continuous color scale (sampling date) and convey this by a blending of colors -- the samples which make up this deme actually span Jun 2016 and Oct 2016.

> You can see the legend representing in currently selected metadata in either the top-left of the phylogenetic tree or the top-right of the map (see "3" in Figure 1).

Reviewer 2: ```suggestion
You can see the legend representing currently selected metadata in either the top-left of the phylogenetic tree or the top-right of the map (see "3" in Figure 1).
```

The **location** of circles represents the geographic resolution currently selected.

> Looking at Figure 1, we have a geo-resolution of Country (labelled "5") and so each circle represents an individual country.

Reviewer 2: ```suggestion
Looking at Figure 1, we have a geo-resolution of "Country" (labelled "5") and so each circle represents an individual country.
```

In the [zika dataset](https://nextstrain.org/zika) you can change this to "Region" and you'll see the map update accordingly.

> Each dataset is responsible for associating different locations with a GPS co-ordinate which we use as the center of the circle -- bear this in mind when you are interpreting the map and don't over-assign precision.

Reviewer: ```suggestion
Each dataset is responsible for associating different locations with a GPS co-ordinate which we use as the center of the circle -- bear this in mind when you are reading the map and don't over-interpret the precision.
```

For example, the circle representing the USA (labelled "1" in Figure 1) represents the entire country, not a specific location in Kansas!

## Transmissions (Lines)

These lines typically represent our best estimate of the movement of the pathogen over the selected time slice.

> For instance, the line indicated by "2" in Figure 1 indicates inferred movement of Zika virus from French Polynesia to Brazil.

Reviewer: Not sure I have a solution, but I think the reference to Fig 1, which is now way high up on the page, is confusing, when Fig 3 is just below (and also has a line '2'). Maybe better to just switch to referencing Fig 3 entirely?

> Each individual line represents a parent-child branch in the phylogenetic tree where the value of the current geographic-resolution differs.

Reviewer 2: ```suggestion
Each individual line represents a parent-child branch in the phylogenetic tree where the value of the current geographic resolution differs.
```

> This means that if values for the geographic-resolution haven't been inferred for internal nodes of the tree then no transmission lines can be drawn.

Reviewer 2: ```suggestion
This means that if values for the geographic resolution haven't been inferred for internal nodes of the tree then no transmission lines can be drawn.
```

> There is a toggle to switch these off in the sidebar (Figure 1, label "5"). 

Reviewer: ```suggestion
There is a toggle to switch the transmission lines off in the sidebar (Figure 1, label "5"). 
```

![](./img/labelled-movement.png)
_Figure 3.
Left: Cartoon of how the tree determines the construction of transmission lines.
Right: Example of transmission lines showing [between country movement of Ebola](https://nextstrain.org/ebola)._

**Line thickness.**

> Each rendered arc is actually a collection of individual lines of identical thickness (as above, each line identified by a branch in the tree indicating a change in the value of the current geographic resolution).

Reviewer 2: ```suggestion
Each rendered arc is actually a collection of individual lines of identical thickness; in Figure 3, each line identified by a branch in the tree indicates a change in the value of the current geographic resolution.
```

I'm confused by the parenthetical statement referring to the figure and how it is relating lines in the tree (branches) to lines on the map (or not relating them).

Author: Clarified 👍

Lines which look "thicker" therefore represent more observed cases of movement between the two locations (circles).

> For instance, in figure 3 the arc labelled by "1" is thicker than that labelled "3", indicating that in the [Ebola dataset](https://nextstrain.org/ebola) there are more observed cases of movement between Guinea and Sierra Leone than there are between Liberia and Guinea.

Reviewer 2: ```suggestion
For instance, in Figure 3 the arc labelled "1" is thicker than the arc labelled "3", indicating that in the [Ebola dataset](https://nextstrain.org/ebola) there are more observed cases of movement between Guinea and Sierra Leone than there are between Liberia and Guinea.
```

> **Directionality** is conveyed by the [orientation of the arc](https://en.wikipedia.org/wiki/Curve_orientation).

Reviewer 2: I don't grasp this very well even after reading the wikipedia article. Maybe we can chat on the phone about how you commonly explain this to people orally (e.g. while giving a presentation of nextstrain) and that will help inform any suggestions I would have for this section.

Author: 👍 rewritten this section to use language such as "the inside of the curve is on the left-hand-side"

A transmission line is a positively oriented curve when traveling from source to origin (parent node to child node on the tree). 
In figure 3, line 1 is positively curved when moving from Guinea to Sierra Leone and therefore indicates the inferred viral movements from Guinea to Sierra Leone; line 2 represents movements from Sierra Leone to Guinea.

**Line color** 

> It represents the currently selected "color-by" -- the metadata displayed in the legend -- of each transmission.

Reviewer 2: ```suggestion
The color of transmission lines represents the currently selected "color-by" -- the metadata displayed in the legend -- of each transmission.
```

> If we examine the cartoon example in figure 3, we see that the parent and child nodes have a different location, which we will use to draw a line on the map from location A to location B. If the current color-by is "date", then the line will be colored blue, which corresponds to the metadata at the origin of the transmission.

Reviewer 2: ```suggestion
If we examine the cartoon example in Figure 3, we see that the parent and child nodes have a different location, which we will use to draw a line on the map from location A to location B. The color-by in this example is "date", so the line will be colored blue, which corresponds to the date value at the origin of the transmission.
```
The color helps me understand directionality better than the orientation, for what it's worth.

Edit: reading on, I see why color can't be the cue used for directionality:
> As transmission arcs potentially consist of a number of lines, we can have multiple different colors within a transmission arc between locations.

As transmission arcs potentially consist of a number of lines, we can have multiple different colors within a transmission arc between locations. 
Looking at the transmission from Guinea to Sierra Leone (labelled "1" in Figure 3), we can see that there are some transmissions from earlier in the outbreak (blue colors) as well as a number of transmissions from later on (orange colors).

If the selected color-by is not present at the origin of an individual transmission, then the line will be colored grey.
It's important to realise that the line color is not related to the geographic resolution, which determines the presence and layout of the lines.

**Uncertainty** is always inherent in the reconstructions from which the transmission data is drawn.
Often this uncertainty is encoded in the dataset, and we use this to reduce the saturation of branch colors in the phylogenetic tree.

> Currently we don't convey this uncertainty in the map -- we simply use whatever values the dataset defines for the geographic resolution in question. 

Reviewer: ```suggestion
Currently we don't convey this uncertainty in the map -- we simply use whatever values the dataset defines for the geographic resolution in question. 
Thus, it's a good idea, when reading the map, to check the phylogeny to get an idea of the uncertainty in a transmission line.
```

## Animation

> If we click the "Play" button (Figure 1, label "4") we can watch the outbreak unfold before our eyes. 

Reviewer 2: Small thing but I like this notation and am in favor of adopting it consistently throughout the doc (and others with labelled figures:
> (Figure 1, label "4")

Author: Agreed!
